examples/hyperopt/README.md

```diff
@@ -0,0 +1,147 @@
+## Hyperparameter Optimization
+
+This document provides a walkthrough of the hyperparameter optimization example,
+which consists of three files.
+
+- [driver.py](driver.py)
+- [functions.py](functions.py)
+- [worker.py](worker.py)
+
+To run the application, run the following.
+
+```
+python driver.py
+```
+
+Machine learning algorithms often have a number of *hyperparameters* whose
+values must be chosen by the practitioner. For example, an optimization
+algorithm may have a step size, a decay rate, and a regularization coefficient.
+In a deep network, the network architecture itself (e.g., the number of layers
```

Comment:
Change "architecture" to something else? Like "scale" or "size" or (maybe) "topology"? Architecture immediately makes me think of e.g. "feed-forward NN vs. RNN", not "10-layer-feed-forward vs. 20-layer-feed-forward". To me those are the same architecture, and I'm not sure if that goes against the formal definition but you are nevertheless explaining this to a lay audience...